src/TraceEvent/Computers/StartStopActivityComputer.cs

```diff
@@ -733,6 +735,9 @@ internal static unsafe string CreateActivityPathString(Guid guid)
         private static readonly Guid AdoNetProvider = new Guid("6a4dfe53-eb50-5332-8473-7b7e10a94fd1");
         private static readonly Guid MicrosoftWindowsHttpService = new Guid("dd5ef90a-6398-47a4-ad34-4dcecdef795f");
         private static readonly Guid MicrosoftDiagnosticsDiagnosticSourceProvider = new Guid("ADB401E1-5296-51F8-C125-5FDA75826144");
+        private static readonly Guid MicrosoftApplicationInsighsDataProvider = new Guid("a62adddb-6b4b-519d-7ba1-f983d81623e0");
+
+        private static readonly Guid GUID_ONE = new Guid("00000001-0000-0000-0000-000000000000");
```

Comment:
Do we know when this guid gets generated?   I would like to know more about it (we should probably put it in a comment here.